phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
+.smiley-box .smiley-list-item {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-list-item .smiley-add {
+	padding: 0;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
+.button .dropdown .pointer-inner {
+	top: 1px;
+}
+
```

Comment:
maybe because https://github.com/phpbb/phpbb/pull/4787/files#diff-4f99eab50954fb0ca94da0bdd18eb2b3R110? as span is not a block